### PR TITLE
add input for callback receipt

### PR DESCRIPTION
**Pull request by fqliao**
This change is [Reviewable](https://reviewable.io/reviews/fisco-bcos/fisco-bcos/846)

### CODE REVIEW

File: libethcore/Transaction.cpp [hunk 1]
```diff
@@ -292,7 +292,7 @@ void Transaction::triggerRpcCallback(LocalisedTransactionReceipt::Ptr pReceipt)
     try
     {
         if (m_rpcCallback)
-            m_rpcCallback(pReceipt);
+            m_rpcCallback(pReceipt, bytes());
```
Comment (bxq2011hust) [inline libethcore/Transaction.cpp]:
```suggestion
{
            m_rpcCallback(pReceipt, bytes());
  }
```

File: librpc/Rpc.cpp [hunk 2]
```diff
@@ -1006,7 +1006,8 @@ std::string Rpc::sendRawTransaction(int _groupID, const std::string& _rlp)
         if (currentTransactionCallback)
         {
             auto transactionCallback = *currentTransactionCallback;
-            tx.setRpcCallback([transactionCallback](LocalisedTransactionReceipt::Ptr receipt) {
+            tx.setRpcCallback([transactionCallback](
+                                  LocalisedTransactionReceipt::Ptr receipt, bytes const& input) {
```
Comment (bxq2011hust) [inline librpc/Rpc.cpp]:
```suggestion
                                  LocalisedTransactionReceipt::Ptr receipt, bytes input) {
```

Comment (wheatli) [reply librpc/Rpc.cpp]: @bxq2011hust why do you think can not use const& here ?

File: libethcore/Transaction.cpp (hunk 1 above)
Comment (fqliao) [reply libethcore/Transaction.cpp]: added { }

File: librpc/Rpc.cpp (hunk 2 above)
Comment (bxq2011hust) [reply librpc/Rpc.cpp]: > @bxq2011hust why do you think can not use const& here ?
because this is async call

Comment (fqliao) [reply librpc/Rpc.cpp]: Ok, I removed the const& for the input parameter.

**Pull request merged**